lib/salus/processor.rb

```diff
@@ -113,6 +113,31 @@ def scan_project
       end
     end
 
+    def create_full_sarif_diff(sarif_diff_full)
+      sarif_file_new, sarif_file_old = sarif_diff_full.split(" ")
```

Comment:
I'm okay with this... but curious does thor support passing them without manually wrapping in quotes?

Reply:
Yes, thor supports it.  
I updated my code to work without quotes.